controllers/remove_process_groups.go

```diff
@@ -150,6 +151,7 @@ func removeProcessGroup(ctx context.Context, r *FoundationDBClusterReconciler, c
 		return err
 	}
 	if len(pvcs.Items) == 1 && pvcs.Items[0].DeletionTimestamp.IsZero() {
+		logr.FromContextOrDiscard(ctx).V(1).Info("Deleting pvc", "name", pvcs.Items[0].Name)
```

Comment:
You missed the same log message above for deleting a Pod.

Reply:
The above method is calling `DeletePod` and I already added logger there https://github.com/FoundationDB/fdb-kubernetes-operator/blob/9a2fe7f545f7b4e524baea0072fc05868bffeb74/pkg/podmanager/pod_lifecycle_manager.go#L110